packages/frontend/src/components/UserSettingsModal/InvitesPanel.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { FC } from 'react';
+import { formatTimestamp } from 'common';
+import { Button, FormGroup, InputGroup, Callout } from '@blueprintjs/core';
+import CopyToClipboard from 'react-copy-to-clipboard';
+import { useApp } from '../../providers/AppProvider';
+import {
+    useCreateInviteLinkMutation,
+    useRevokeInvitesMutation,
+} from '../../hooks/useInviteLink';
+import { useTracking } from '../../providers/TrackingProvider';
+import { EventName } from '../../types/Events';
+
+const InvitePanel: FC = () => {
+    const { track } = useTracking();
+    const { showToastSuccess } = useApp();
+    const inviteLink = useCreateInviteLinkMutation();
+    const revokeInvitesMutation = useRevokeInvitesMutation();
+
+    return (
+        <div
+            style={{ height: '100%', display: 'flex', flexDirection: 'column' }}
+        >
+            <FormGroup
+                label="Invite users to your organization"
+                labelFor="invite-link-input"
+            >
+                {inviteLink.data ? (
+                    <>
+                        <InputGroup
+                            id="invite-link-input"
+                            type="text"
+                            readOnly
+                            value={`${window.location.protocol}//${window.location.host}/invite/${inviteLink.data.inviteCode}`}
+                            rightElement={
+                                <CopyToClipboard
+                                    text={`${window.location.protocol}//${window.location.host}/invite/${inviteLink.data.inviteCode}`}
+                                    options={{ message: 'Copied' }}
+                                    onCopy={() =>
+                                        showToastSuccess({
+                                            title: 'Invite link copied',
+                                        })
+                                    }
+                                >
+                                    <Button minimal icon="clipboard" />
+                                </CopyToClipboard>
+                            }
+                        />
+                        <Callout intent="primary" style={{ marginTop: 10 }}>
+                            Share this link with your colleagues and they can
+                            join your organization. This link will expire at{' '}
```

Comment:
Missing var?

Reply:
No, it's a common hack to force a white space between the text and the following element. In fact, it was our prettier that added it on pre-commit.